### PR TITLE
Add Tilt and Skaffold.

### LINKED ISSUE #5: [FEATURE] add k8s CD tools
Hi,

I have a feature request.
I wonder if it's possible to add CD tools for Kubernetes like [Tilt](https://tilt.dev) and [Skaffold](https://skaffold.dev)?

Releases can be found here.
https://github.com/tilt-dev/tilt/releases
https://github.com/GoogleContainerTools/skaffold/releases

In order to reduce footprint it would also be nice if they were made them optional.
Where this would not install the binaries.
```yaml
    - uses: yokawasa/action-setup-kube-tools@v0.3.0
      with:
        tilt: null
        skaffold: null
```

And  this would install them.
```yaml
    - uses: yokawasa/action-setup-kube-tools@v0.3.0
      with:
        tilt: '0.18.11'
        skaffold: '1.20.0'
```

A side benefit of this is also that there is no need to track external versions in this project, simply default to null and only install them when the user sets the version.

### PR DESCRIPTION
**Pull request by Raboo**
Add Tilt and Skaffold.
Minor markdown cleanup.
Bump actions/checkout to v2.

Partially fixes #5. It adds Tilt and Skaffold, but this does not make them optional.
I'm not really a javascript/typescript coder, so I basically did a copy-paste from when you added Rancher CLI.
I haven't really tested this, I'm hoping that this PR will do that for me.

### CODE REVIEW

**Comment by Raboo** [comment]
Seems like specifying version doesn't work. But it doesn't feel like that's something I broke?

**Comment by yokawasa** [comment]
@Raboo Thank you for the PR. I'll take a look!

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -52,7 +56,9 @@ jobs:
         kubeval: '0.14.0'
         conftest: '0.18.2'
         rancher: '2.4.10'
-      id: setup
+        tilt: '0.18.11'
+        skaffold: '1.20.0'
+        id: setup
```
Comment (yokawasa) [inline .github/workflows/test.yml:61]:
```suggestion
      id: setup
```

**Review by yokawasa — COMMENTED** [review]
I added one suggestion which I believe resolve the GitHub Action failure.

**Comment by Raboo** [comment]
Ah, damn vscode auto indentation... I've fixed it now.

**Review by yokawasa — APPROVED** [review]
lgtm!!

**Comment by Raboo** [comment]
Alright nice, so when does this get merged?

**Comment by yokawasa** [comment]
@Raboo looks everything good! I merge now

**Comment by yokawasa** [comment]
@Raboo 
https://github.com/yokawasa/action-setup-kube-tools/releases/tag/v0.4.0
Thanks alot!!

**Pull request merged**